x/programs/rust/examples/counter/Cargo.toml

```diff
@@ -7,6 +7,7 @@ edition = "2021"
 
 [dependencies]
 wasmlanche-sdk = { version = "0.1.0", path = "../../wasmlanche-sdk" }
+borsh = { version = "1.5.0", features = ["derive"] }
```

Comment:
I think I need to put this in a peer dependency list or re-export

Reply:
Could you please expand on what is a peer dependency and why we would want this please ?

Reply:
Ignore me